source/testcommon/testcommon/get_then_names.py

```diff
@@ -0,0 +1,8 @@
+import inspect
+from pathlib import Path
+
+
+def get_then_names() -> list[str]:
```

Comment:
Let's add docstrings to all the public functions. It provides a much better DX for consumers.